.github/workflows/labeler.yml

```diff
@@ -29,3 +28,39 @@ jobs:
     - uses: actions/labeler@v5
       with:
         configuration-path: .github/configs/labeler.yml
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: Compare trunk and PR
+      env:
+        GH_TOKEN: ${{ github.token }}
+      run: |
+        BASE_COMMIT="${{ github.event.pull_request.base.sha }}"
+        HEAD_COMMIT="${{ github.event.pull_request.head.sha }}"
+
+        summary=$(git diff --stat $BASE_COMMIT $HEAD_COMMIT | tail -n 1)
```

Comment:
Not sure if it helps, but we can use the `gh` CLI here. 

https://cli.github.com/manual/gh_pr_diff

Reply:
Thanks. It is very useful.